Python/Product/EnvironmentsList/Resources.resx

```diff
@@ -422,6 +422,12 @@ The folder will be deleted from disk. This operation cannot be undone.</value>
   <data name="PipExtensionClearSearchToolTip" xml:space="preserve">
     <value>Clear search filter</value>
   </data>
+  <data name="PipExtensionSecurityWarning" xml:space="preserve">
+    <value>Due to security issues, pip may no longer work on this version of Python.</value>
```

Comment:
Rather than "issues", let's say "Due to new security restrictions, installing from the internet may not work ..."

Reply:
Agreed I like that better.